src/components/custringload.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
```

Comment:
I think this file is duplicated

Reply:
hi joao

I deleted this file before committing and pushing. so it is not present locally . can you please suggest what can i do

Reply:
oh i actually didn't delete it . i renamed it

Reply:
ok done. hopefully i get it correct this time